src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -223,35 +266,15 @@ trait EntityComponent {
 
     /** list all entities of the given type in the workspace */
     def list(workspaceContext: SlickWorkspaceContext, entityType: String): ReadAction[TraversableOnce[Entity]] = {
-      val sql = sql"""#$baseEntityAndAttributeSql where e.entity_type = ${entityType} and e.workspace_id = ${workspaceContext.workspaceId}""".as[EntityListResult]
-      unmarshalEntities(sql)
+      unmarshalEntities(EntityAndAttributesRawSqlQuery.actionForType(workspaceContext, entityType))
     }
 
     def list(workspaceContext: SlickWorkspaceContext, entityRefs: Traversable[AttributeEntityReference]): ReadAction[TraversableOnce[Entity]] = {
-      val baseSelect = sql"""#$baseEntityAndAttributeSql where e.workspace_id = ${workspaceContext.workspaceId} and (e.entity_type, e.name) in ("""
-      val entityTypeNameTuples = reduceSqlActionsWithDelim(entityRefs.map { ref => sql"(${ref.entityType}, ${ref.entityName})" }.toSeq)
-      unmarshalEntities(concatSqlActions(baseSelect, entityTypeNameTuples, sql")").as[EntityListResult])
+      unmarshalEntities(EntityAndAttributesRawSqlQuery.actionForRefs(workspaceContext, entityRefs))
     }
 
     def listByIds(entityIds: Traversable[Long]): ReadAction[Map[Long, Entity]] = {
-      val baseSelect = sql"""#$baseEntityAndAttributeSql where e.id  in ("""
-      val entityIdSql = reduceSqlActionsWithDelim(entityIds.map { id => sql"$id" }.toSeq)
-      unmarshalEntitiesWithIds(concatSqlActions(baseSelect, entityIdSql, sql")").as[EntityListResult])
-    }
-
-    /**
-     * Extends given query to query for attributes (if they exist) and entity references (if they exist).
-     * query joinLeft entityAttributeQuery join attributeQuery joinLeft entityQuery
-     * @param query
-     * @return
-     */
-    def joinOnAttributesAndRefs(query: EntityQuery): EntityQueryWithAttributesAndRefs = {
```

Comment:
it wasn't so I removed it because it uses a joinLeft... it was probably left over from before we started with raw sql